components/Footer.js

```diff
@@ -1,7 +1,81 @@
 import React from "react";
 
 const Footer = () => {
-	return <div>Footer</div>;
+	return (
+		<footer className='dark:bg-neutral-900 dark:text-gray-50 py-6'>
+			<div className='container p-6 mx-auto lg:space-y-8'>
+				<h2 className='text-4xl font-lexend font-bold text-center'>
+					Let&#39;s connect!
+				</h2>
+				<div className='flex flex-wrap justify-center justify-around px-80'>
+					<a
+						rel='noopener noreferrer'
+						href='https://www.instagram.com/acm_ucr/'
+						target='_blank'
+						title='Instagram'
+						className='w-12 h-12 mx-10 my-6 fill-current md:mx-12 lg:m-0 dark:text-gray-50 hover:text-purple-500'
```

Comment:
`w-12` and `h-12` are also dangerous values to be working with and the margin levels as well. Use percentages when possible so its scales nicely, same applies for margins. 

Margins are not enough (to replicate, just try resizing your window): 
<img width="1025" alt="image" src="https://user-images.githubusercontent.com/36285610/184803706-59bf5ff3-ee08-46f9-a09a-8657bc85586a.png">

Breaks into Column Very Quickly:
<img width="936" alt="image" src="https://user-images.githubusercontent.com/36285610/184803798-eb3a4cf2-2f50-4111-9e16-3672636d80a4.png">